Gruntfile.js

```diff
@@ -88,18 +88,34 @@ module.exports = function(grunt) {
 			dryrun: { options: {service: "qa", dryRun: true} },
 			qa: { options: {service: "qa"} },
 			prod: { options: {service: "prod"} }
+		},
+		"shell": {
+			"deployrumlambda": {
+				command: env => {
+
+					// These AWS profiles are expected to be in ~/.aws/credentials
```

Comment:
Yes, they're in Lastpass - created by Cloud enablement team today, and configuring them is documented in the changes I made to README.md.  I've also added a TODO to the PR OP to share the lastpass secret appropriately.